bundles/org.openhab.binding.openwebnet/src/main/resources/OH-INF/thing/BusEnergyMeter.xml

```diff
@@ -31,6 +33,12 @@
 				<description>A "5" must be added before the energy meter address: 5N with N=[1-255]. Example energy meter "42" -->
 					where="542"</description>
 			</parameter>
+
+			<parameter name="energyRefreshPeriod" type="integer" min="30" max="1440" required="true">
+				<label>Energy Totalizers Refresh Period</label>
+				<description> Sets the number of minutes between refreshes for energy totalizers (default: 30 minutes)</description>
```

Comment:
Minor: please remove the leading space

Reply:
Done

Reply:
@fabgio why do you mark the conversation as "resolved" and comment "Done" when there is no commit that implements the requested change yet? I think this is very confusing...

Reply:
ok

Reply:
Not done, the initial space is still there.

Reply:
ok

Reply:
Maybe I am not clear.
Please replace "<description> Sets ..." byt "<description>Sets ..."
Just one space to remove